Server/pom.xml

```diff
@@ -14,23 +14,11 @@
 
     <modelVersion>4.0.0</modelVersion>
 
-
-    <repositories>
-        <repository>
-            <id>local-maven-repo</id>
-            <url>file:///${project.basedir}\local-maven-repo</url>
-        </repository>
-    </repositories>
-
```

Comment:
Where you able to resolve the issue on your local project? I was able to successfully fix the dependencies on my IDE without the need for the local-maven-repo. Please reach out if you need help.

Reply:
I think the problem with removing the local maven repo is that the wek Deepl 1.7.0 dependency isn't resolvable online. This is why we need a local maven repo in the first place. However, since you installed that repo on your computer, this dependency is resolved. However, I don't think this would work on someone who clones the repo for the first time. @Khal-Shah could you have a look at this please?

Reply:
As mentioned by @JayChouteau, the local maven repo is needed to hold the wekaDeeplearning4j-1.7.0 jar which is utilized to build the LSTM model. This jar file is a plugin for the weka library that isn't available in the central maven repository.

In our unit test folder, you can try to run the LSTMTest and you'll most likely get errors because a lot of dependencies will be missing. Please reinstate it.

Reply:
I added the local repository back. Maybe Khal and other people could give a look at this as well.